docker/run.bash

```diff
@@ -20,45 +20,80 @@
 # Runs a docker container with the image created by build.bash
 # Requires:
 #   docker
-#   nvidia-docker
 #   an X server
-#   rocker
+# Optional:
+#   A joystick mounted to /dev/input/js0 or /dev/input/js1
 
-# Display Help
-Help()
-{
-   echo "Runs a docker container with the image created by build.bash."
-   echo
-   echo "Syntax: script [-d|s] image_name"
-   echo "options:"
-   echo "-d     Use for development with host system volume mount."
-   echo "-s     Simulation purposes only."
-   echo
-}
-
-if [ $# -lt 2 ]
+if [ $# -lt 1 ]
 then
-    Help
+    echo "Usage: $0 <docker image> [<dir with workspace> ...]"
     exit 1
 fi
 
-while getopts ":ds" option; do
-  case $option in
-    d) # Build image for development
-      echo "Building Development image"
-      ROCKER_ARGS="--devices /dev/dri/ --dev-helpers --nvidia --x11 --user --home --git ";;
-    s) # Build image for Simulation
-      echo "Building Simulation image"
-      ROCKER_ARGS="--devices /dev/dri/ --dev-helpers --nvidia --x11 --user";;
-  esac
-done
+DOCKER_OPTS=
+
+IMG=$(basename $1)
+
+ARGS=("$@")
+WORKSPACES=("${ARGS[@]:1}")
```

Comment:
I didn't get this part here, what should I input this argument, `buoys_ws`?

Reply:
This variable is automatically assigned. The command to run should just be `./run.bash buoy`